### ISSUE TITLE
能单独给出一个baseline的代码吗？

### ISSUE THREAD

**Issue by ykding666** [BoomShakaY/AIM-CCReID#9]
我注意到您论文中baseline的效果很好，我也只使用resnet，id loss。却跑不出像您那样的结果，尤其是ltcc数据集

**Comment by BoomShakaY** [comment]
The baseline code is accessible at [here](https://github.com/guxinqian/Simple-CCReID), you can directly pull their code and try to reproduce the results. However, the performance of the baseline method may vary depending on the server, environment, hyperparameters, etc. You can also take a look at the issues under their repo, I hope it can solve your concern.

**Comment by raisa1921** [comment]
Hello sir, fusing features from two encoders produced **NaN** losses. Did that happen to you too? Also, can you please tell me at which epoch the adversarial loss should be set and should I use the full weight (i.e., 1) of adversarial loss for PRCC dataset? I got accuracies nearly 50% (highest 49.40% R@1 and 50.7% mAP) for PRCC with Cloth-Changing setup.
Sir, what might be the reason for such a large gap?

**Comment by BoomShakaY** [comment]
Sorry for the late reply, I did encounter such NaN situations in my training procedure. Referring to the baseline code, the adversarial loss may be set after the first time decreasing the learning rate (which may be 25), but you may modify this parameter to find the optimal one for your model. 

I think the large gap in accuracy might occur with the NaN loss.

**Comment by BoomShakaY** [comment]
@raisa1921 If you have further questions you can open a new issue, I currently let this issue close.
